pkg/build/api/types.go

```diff
@@ -16,6 +16,9 @@ type Build struct {
 	// Status is the current status of the build.
 	Status BuildStatus `json:"status,omitempty" yaml:"status,omitempty"`
 
+	// A human readable message indicating details about why the build has this status
+	Message string `json:"message,omitempty"`
```

Comment:
See upstream, yaml is nuked, I'm tired of copying it.

----- Original Message -----
> > @@ -16,6 +16,9 @@ type Build struct {
> >  	// Status is the current status of the build.
> >  	Status BuildStatus `json:"status,omitempty" yaml:"status,omitempty"`
> >  
> > +	// A human readable message indicating details about why the build has
> > this status
> > +	Message string `json:"message,omitempty"`
> 
> how about yaml?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/679/files#r23208028